src/tusb_option.h

```diff
@@ -59,6 +59,7 @@
 #define OPT_MCU_SAMD21            200 ///< MicroChip SAMD21
 #define OPT_MCU_SAMD51            201 ///< MicroChip SAMD51
 #define OPT_MCU_SAME5X            203 ///< MicroChip SAM E5x
+#define OPT_MCU_SAME51            205 ///< MicroChip SAME51
```

Comment:
This is because in hw/bsp/board_mcu.h there is an extra define for the SVC_Handler handler. While the code will compile fine without it, the binary won't run on the board. Since I don't have any other SAMX5X devices to test, I thought it prudent to add the extra define.

Reply:
This is application/bsp related issue. The tinyusb stack doesn't need to know this difference. This should be resolved at  application/bsp level. You could use the macro `__SAME51J19A__` or adding bsp own `__SAME51__` to the bsp to resolve this.